pkg/kubelet/container/runtime.go

```diff
@@ -57,7 +57,10 @@ type Runtime interface {
 	Version() (Version, error)
 	// APIVersion returns the API version information of the container
 	// runtime. This may be different from the runtime engine's version.
+	// TODO(random-liu): We should fold this into Version()
 	APIVersion() (Version, error)
+	// CheckStatus checks whether the underlying runtime is working normally.
+	CheckStatus() error
```

Comment:
I prefer `Status` over `CheckStatus`. The `Check` IMHO seems redundant.
The description should say `Returns error if the runtime is unhealthy; nil otherwise.`